packaging/installer/kickstart.sh

```diff
@@ -293,6 +299,35 @@ while [ -n "${1}" ]; do
 	elif [ "${1}" = "--stable-channel" ]; then
 		RELEASE_CHANNEL="stable"
 		shift 1
+	elif [ "${1}" == "--local-tarball-override" ]; then
+		shift 1
+		if [ -z "${1}" ]; then
+			fatal "Missing netdata: Option --local-tarball-override requires extra information. The desired tarball for netdata, the checksumi, the go.d plugin tarball and the go.d plugin config tarball, in this particular order"
```

Comment:
it is hard to understand the order when it is one line with `,` and `and`
Maybe numbered list?

Reply:
The error on Debian 9.9 is different, this time it did not showed the error that I had on RHEL 7, but it did not stop the installation process, it began to compile with only one argument given:

![VirtualBox_Debian9_21_08_2019_11_52_53](https://user-images.githubusercontent.com/49162938/63429778-85803600-c40a-11e9-8378-d67017014a8b.png)

Reply:
wow, this is interesting @thiagoftsm, you need to provide me with the BASH versions from both cases

@ilyam8 adding comma separated list of items does not provide some value. space or comma, they are still an ordered list of items that the user must provide in the right order.  If we add `,` for separation, it will just make us do more processing to get what we want and validate

Reply:
On RHEL 7.7 I am using bash version bash-4.2.46-33.el7 .
On Debian 9.9 I am using bash version bash-4.4-5.
Let me know case you need more information.

Reply:
ah, spotted the error for that one. My apologies, it should be single equality rather than double